build.xml

```diff
@@ -70,7 +70,7 @@
     <!-- Behat -->
     <target name="test:behat" depends="override:property:APP_ENV, cache:clear, database:drop, database:create, database:migrate, database:fixture:test" description="Executes behat tests." >
         <echo msg="Running behat tests..."/>
-        <exec command="APP_ENV=${APP_ENV} APP_URL=http://localhost:8001 php -d error_reporting='E_ALL &amp; ~E_DEPRECATED' bin/behat --colors --stop-on-failure --format=progress"
+        <exec command="APP_ENV=${APP_ENV} APP_URL=http://localhost:8001 php -d error_reporting='E_ALL &amp; ~E_DEPRECATED' bin/behat --colors --stop-on-failure --format=progress -vv"
```

Comment:
Why you add `-vv` for a local test ?

Reply:
Not sure what you're referring to by "local test"? Verbose option for Behat improves verbosity of exceptions. It's quite difficult to figure out the exception just by the message and this attaches the stack trace to the output.